packages/test/test-service-load/src/runner.ts

```diff
@@ -324,6 +337,78 @@ function scheduleContainerClose(
     });
 }
 
+async function setupOpsMetrics(container: IContainer, logger: ITelemetryLogger, progressIntervalMs: number) {
+    // Use map to cache userName instead of recomputing.
+    const clientIdUserNameMap: { [clientId: string]: string } = {};
+
+    const getUserName = (userContainer: IContainer) => {
+        const clientId = userContainer.clientId;
+        if (clientId !== undefined && clientId.length > 0) {
+            if (clientIdUserNameMap[clientId]) {
+                return clientIdUserNameMap[clientId];
+            }
+
+            const userName: string | undefined = userContainer.getQuorum().getMember(clientId)?.client.user.id;
+            if (userName !== undefined && userName.length > 0) {
+                clientIdUserNameMap[clientId] = userName;
+                return userName;
+            }
+        } else {
+            return "Unknown";
+        }
+    };
+
+    let submitedOps = 0;
+    container.deltaManager.on("submitOp", (message) => {
```

Comment:
Note - this will closely, but not accurately represent number of ops sent by this client and sequenced, as some ops will get lost due to connection closure. It's probably small diff, but do not be surprised if numbers do not add up somewhere.

Reply:
We use these numbers for pattern and within error, there are multiple reasons for which its hard to reconcile it to zero error.